python/hail/expr/functions.py

```diff
@@ -2440,7 +2439,10 @@ def len(x):
     -------
     :class:`.Expression` of type :class:`.TInt32`
     """
-    return x._method("size", tint32)
+    if isinstance(x.dtype, TTuple):
```

Comment:
use lower case ttuple here. It'll work and won't cause problems with what I'm working on now.